content/rippled-api-methods/account_offers.md

```diff
@@ -163,8 +163,8 @@ The response follows the [standard format](#response-formatting), with a success
 | `offers`               | Array                                      | Array of objects, where each object represents an offer made by this account that is outstanding as of the requested ledger version. If the number of offers is large, only returns up to `limit` at a time. |
 | `ledger_current_index` | Integer                                    | (Omitted if `ledger_hash` or `ledger_index` provided) Sequence number of the ledger version used when retrieving this data. [New in: rippled 0.26.4-sp1][] |
 | `ledger_index`         | Integer                                    | (Omitted if `ledger_current_index` provided instead) Sequence number, provided in the request, of the ledger version that was used when retrieving this data. [New in: rippled 0.26.4-sp1][] |
-| `ledger_hash`          | String                                     | (May be omitted) Hex hash, provided in the request, of the ledger version that was used when retrieving this data. [New in: rippled 0.26.4-sp1][] |
-| `marker`               | [(Not Specified)](#markers-and-pagination) | Server-defined value indicating the response is paginated. Pass this to the next call to resume where this call left off. Omitted when there are no pages of information after this one. [New in: rippled 0.26.4][] |
+| `ledger_hash`          | String                                     | _(May be omitted)_ Hex hash, provided in the request, of the ledger version that was used when retrieving this data. [New in: rippled 0.26.4-sp1][] |
```

Comment:
Why is `(May be omitted)` italicized?

I see the phrase used a few other places ([example](https://github.com/wilsonianb/ripple-dev-portal/blob/c3124b6991617257b1c98c358715a590ec150eb2/content/rippled-api-methods/account_lines.md)) where it's not in italics.

Reply:
The example(s) I looked at had it italicized, but it does look like more are not.
@mDuo13 Which side do you lean on?

Reply:
Italicize all the parentheticals! Well, at least all the ones that start a Description column.

(Some of the older stuff pre-dates this rule. Feel free to update cases you see that aren't italicized.)